pkgs/development/python-modules/backports_cached-property/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder
+, setuptools, setuptools-scm, wheel
+}:
+
+buildPythonPackage rec {
+  pname = "backports.cached-property";
+  version = "1.0.2";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-kwb57tbsVf0Vas5rwQlOLIb65fsr8HtqnAB0XGVudd0=";
+  };
+
+  nativeBuildInputs = [
+    setuptools setuptools-scm wheel
+  ];
```

Comment:
Don`t need move to `propagatedBuildInputs` section?

Reply:
Depends if setuptools is a runtime dependency. wheel should already be there.

Reply:
Updated PR.